nautobot/core/api/renderers.py

```diff
@@ -140,18 +140,22 @@ def object_to_row_elements(self, record, *, headers):
                 elif "value" in value and "label" in value:
                     # An enum type
                     value = value["value"]
+                elif "id" in value:
+                    value == str(value["id"])
                 else:
                     value = json.dumps(value)
             elif isinstance(value, (list, tuple, set)):
                 if isinstance(value, set):
                     value = sorted(value)
-                if value and isinstance(value[0], dict) and "composite_key" in value[0]:
+                if value and isinstance(value[0], dict) and ("composite_key" in value[0] or "id" in value[0]):
                     # Multiple nested related objects
                     if value[0].get("generic_foreign_key"):
                         # Multiple *generic* nested related obects
                         value = [COMPOSITE_KEY_SEPARATOR.join([v["object_type"], v["composite_key"]]) for v in value]
-                    else:
+                    elif value[0].get("composite_key"):
```

Comment:
Should composite key be removed from CSV in this PR?

Reply:
Yes, I believe so.

Reply:
Can/should we remove this at this time?